src/bacnet/basic/object/ai.c

```diff
@@ -43,7 +43,7 @@
 #include "bacnet/basic/object/ai.h"
 
 #ifndef MAX_ANALOG_INPUTS
-#define MAX_ANALOG_INPUTS 4
+#define MAX_ANALOG_INPUTS 5000
```

Comment:
could you keep this as 4 just now as we dont want the stack including 5000 instances when we use this build of the stack for the device service

Hopefully if you figure out how to size dynamically them we can do that instead